webapp/src/webapp/core.cljs

```diff
@@ -18,8 +18,19 @@
     (rdom/unmount-component-at-node root-el)
     (rdom/render [views/main-panel] root-el)))
 
+(defn add-global-keyboard-shortcuts []
+  (.addEventListener js/document "keydown" 
+    (fn [e]
+      (when (and (or (.-metaKey e) (.-ctrlKey e)) 
+                 (= (.-key e) "k"))
+        (.preventDefault e)
+        (let [search-input (.getElementById js/document "header-search")]
+          (when search-input
+            (.focus search-input)))))))
+
```

Comment:
The ideal would be to place this listener inside the Editor page and have it remove the listener when you leave the editor page. In other words, when you mount the component, you activate the listener; when you unmount it, you remove the listener.
It is not a good practice to leave it open forever.